_posts/2019-09-04-pulp-3-performance-blog.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Pulp 3 Performance Blog
+author: Imaanpreet Kaur
+team: Red Hat Performance Team
+members: Pradeep Surisetty, Jan Hutař, Imaanpreet Kaur
+---
+
+## Motivation
+
+As we all know, Pulp3 is under active development. So, it's predominant  to check the performance of the pulp3. Pulp team come up with a test plan and Red Hat Performance team tested these test cases. So, let’s discuss about the pulp3 performance in brief.
+
+## Test environment
+
+Pulp was installed on one CentOS7 (CentOS Linux release 7.5.1804 (Core)) system using this [playbook](https://github.com/pulp/ansible-pulp/blob/master/example-use/playbook.yml). 
+
+##### pulp version
+
+![version](https://user-images.githubusercontent.com/32102000/64325263-1a1c9380-cfc8-11e9-809c-210229c85887.png)
```

Comment:
Can we list the plugins and their versions tested in a table instead of the graphic. When rendered the graphic is too dense to be easily read.